demos/gfql/ten_mins_to_gfql.ipynb

```diff
@@ -0,0 +1,2376 @@
+{
```

Comment:
Thank you for this notebook!

* link in here to expose in the nav: https://pygraphistry.readthedocs.io/en/latest/notebooks/gfql.html
* maybe rename from '10min to gfql' to something else, as '10 min to gfql' is already used here: https://pygraphistry.readthedocs.io/en/latest/gfql/about.html
* maybe we put into the fraud demo folder? https://github.com/graphistry/pygraphistry/tree/650ad9b314dca5eae4fc4962c1caa2e97c424fa4/demos/demos_by_use_case/fraud